integration_tests/helpers/walletProcess.js

```diff
@@ -278,22 +280,26 @@ class WalletProcess {
       ".",
       "--password",
       "kensentme",
-      "--command",
-      command,
       "--non-interactive",
       "--network",
       "localnet",
     ];
     if (this.logFilePath) {
       args.push("--log-config", this.logFilePath);
     }
+
     const overrides = this.getOverrides();
     Object.keys(overrides).forEach((k) => {
       args.push("-p");
       args.push(`${k}=${overrides[k]}`);
     });
+
+    // In the new cli commands are just arguments on the console wallet
```

Comment:
nit: This comment probably won't make sense in a few weeks
```suggestion
    // Add command arguments 
```